GitUI/Theming/Renderers/TabRenderer.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Forms;
+using GitExtUtils.GitUI.Theming;
+
+namespace GitUI.Theming
+{
+    internal class TabRenderer : ThemeRenderer
+    {
+        protected override string Clsid { get; } = "Tab";
+
+        public override int RenderTextEx(IntPtr htheme, IntPtr hdc, int partid, int stateid, string psztext, int cchtext, NativeMethods.DT dwtextflags, IntPtr prect, ref NativeMethods.DTTOPTS poptions)
+        {
+            // do not render, just modify text color
+            var textColor = GetTextColor((States)stateid);
+            poptions.crText = ColorTranslator.ToWin32(textColor);
+            poptions.dwFlags |= NativeMethods.DTT.TextColor;
+
+            // proceed to default implementation with modified poptions parameter
+            return Unhandled;
+        }
+
+        public override int RenderBackground(IntPtr hdc, int partid, int stateid, Rectangle prect, NativeMethods.RECTCLS pcliprect)
+        {
+            using var ctx = CreateRenderContext(hdc, pcliprect);
+            switch ((Parts)partid)
+            {
+                case Parts.TABP_TOPTABITEM:
+                case Parts.TABP_TOPTABITEMRIGHTEDGE:
+                case Parts.TABP_TOPTABITEMLEFTEDGE:
+                case Parts.TABP_TOPTABITEMBOTHTEDGE:
+                    RenderTab((Parts)partid, (States)stateid, prect, ctx);
+                    return Handled;
+
+                case Parts.TABP_PANE:
+                    RenderPane(ctx, prect);
+                    return Handled;
+
+                // case Parts.TABP_TABITEM:
+                // case Parts.TABP_TABITEMRIGHTEDGE:
+                // case Parts.TABP_TABITEMLEFTEDGE:
+                // case Parts.TABP_TABITEMBOTHTEDGE:
+                // case Parts.TABP_BODY:
+                // case Parts.TABP_AEROWIZARDBODY:
+                default:
+                    return Unhandled;
+            }
+        }
+
+        private static void RenderPane(Context ctx, Rectangle prect)
+        {
+            var borderPen = GetBorderPen();
+            ctx.Graphics.DrawRectangle(borderPen, prect.Inclusive());
+        }
+
+        private static void RenderTab(Parts partId, States stateId, Rectangle prect, Context ctx)
+        {
+            Brush backBrush = GetTabBackgroundBrush(stateId);
```

Comment:
Doesn't it need to be disposed of?

Reply:
Here we use on of SystemBrushes, since we do not create it, we should not dispose.